kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/resources/jpa/META-INF/persistence.xml

```diff
@@ -94,7 +94,7 @@
 
       <!-- BZ 841786: AS7/EAP 6/Hib 4 uses new (sequence) generators which seem to cause problems -->
       <property name="hibernate.id.new_generator_mappings" value="false" />
-      <property name="hibernate.transaction.jta.platform" value="org.hibernate.service.jta.platform.internal.JBossAppServerJtaPlatform" />
+      <property name="hibernate.transaction.jta.platform" value="JBossAS" />
```

Comment:
so what should it be - JBossAS or JBossTS?

Reply:
It depends on usage.
If it is used within WildFly/EAP then JBossAS should be used (replacement of JBossAppServerJtaPlatform).
If it is used standalone then JBossTS should be used (replacement of JBossStandAloneJtaPlatform).
See https://github.com/hibernate/hibernate-orm/blob/5.1/documentation/src/main/asciidoc/userguide/chapters/transactions/Transactions.adoc

Reply:
ok, thanks for explanation